src/components/React/ControlsPanel.js

```diff
@@ -1,74 +1,82 @@
-import React, {Component} from 'react';
-import Flexbox from 'flexbox-react';
-import { connect } from 'react-redux';
+import React, {useState} from 'react';
 import {Sidebar} from 'primereact/sidebar';
-import {Button} from 'primereact/button';
-import {SelectButton} from 'primereact/selectbutton';
+import {Col, Row, Button, Container, ToggleButtonGroup, ToggleButton} from 'react-bootstrap';
 
-import {myChangeVolumeAction} from '../../redux/AppActions'
 import Controls from './Controls';
+import { useVolume } from '../../context/volume-context';
 
-const options = [
-  {value: './assets/models/48hr_20x_23_0.597976_ 0.597976_5.png:false', label: 'Spheroid old' },
-  {value: './assets/models/r06c03f04_49_1.29_1.29_5.00000.png:false', label: 'Spheroid 1 cropped slices' },
-  {value: './assets/models/r06c03f04_90_1.935_1.935_5.00000.png:false', label: 'Spheroid 1 90 slices' },
-  {value: './assets/models/r06c03f03_49_1.29_1.29_5.00000.png:false', label: 'Spheroid 2 cropped slices' },
-  {value: './assets/models/r06c03f03_90_1.935_1.935_5.00000.png:false', label: 'Spheroid 2 90 slices' },
-];
+export default function ControlPanel(props) {
+  const [sidebarVisible, setSidebarVisible] = useState(false);
 
-export default connect(null, {myChangeVolumeAction})
-( class ControlPanel extends Component{
+  const {
+    state: {toggles},
+    dispatch
+  } = useVolume();
 
-  constructor(props) {
-    super(props);
-    this.state = {
-      sideBarVisible: false,
-      currentData: null,
-      currentVolume:"",
-    };
-    this.volumeSelectChanged = this.volumeSelectChanged.bind(this);
-  }
+  return (
+    <Container fluid className="my-3">
+      <Row>
+        <Col className="text-center">
+          <Button onClick={(e) => setSidebarVisible(!sidebarVisible)}>
+            Options
+          </Button>
+        </Col>
 
-  volumeSelectChanged = (selected) => {
-    this.setState({
-      currentVolume: selected.value,
-      currentData:selected
-    });
-    
-    if(selected.value != null) {
-      var volumeProperties = selected.value.split(":");
-      this.props.myChangeVolumeAction(volumeProperties[0],volumeProperties[1]);
-    } else {
-      this.props.myChangeVolumeAction("", "");
-    }
-  };
+        <Col>
+          <ToggleButtonGroup 
+            type="radio" 
+            name="variable"
```

Comment:
for consistency, I think this name should be measurement

Reply:
We love an extra set of eyes to catch the unfinished name changes haha. All set!